packages/conventional-changelog-writer/test/index.spec.js

```diff
@@ -149,15 +158,17 @@ describe('conventionalChangelogWriter', function () {
 
     it('should not link references', function (done) {
       let i = 0
-
+      const context = {
+        version: '0.5.0',
+        title: 'this is a title',
+        host: 'https://github.com',
+        repository: 'a/b',
+        linkReferences: false
+      }
+      const changelog = conventionalChangelogWriter.parseArray(commits, {})
```

Comment:
This call/test for`.parseArray` looks slightly different than the stream; Should the `context` be passed so they match?